low-level/files/opt/seagate/sspl/conf/sspl.post-install.conf.tmpl

```diff
@@ -0,0 +1,26 @@
+cortx:
+  release:
+    product: LR_R2
```

Comment:
Need setup check key also here.
![image](https://user-images.githubusercontent.com/32053979/111953006-01bd8c80-8b0c-11eb-8e9d-b29f6d2c10bf.png)
Above acitivity was moved to pos-install during review

Reply:
With new commit, setup key is added and available from post-install stage.